.github/workflows/ci.yml

```
@@ -124,7 +124,15 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
+      - name: Set up Go
+        uses: actions/setup-go@v4
```

actionlint (1.7.12)

[error] the runner of "actions/setup-go@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)